ENTITY/SetEntityHealth.md

```diff
@@ -8,17 +8,18 @@ ns: ENTITY
 void SET_ENTITY_HEALTH(Entity entity, int health);
 ```
 
-health >= 0
+When setting health for player ped entity types, the game clamps the health value to ensure it does not exceed the entity's maximum health. This maximum health can be retrieved by invoking [`GET_PED_MAX_HEALTH`](#_0x4700A416E8324EF3). It can also be modified by invoking [`SET_PED_MAX_HEALTH`](#_0xF5F6378C4F3419D3).
 
-male ped ~= 100 - 200
+When setting health for non-player peds, the maximum health will be increased if the new health value exceeds the current maximum, ensuring that the changes take effect.
 
-female ped ~= 0 - 100
+Default health value for peds is `200`.
 
-```
-NativeDB Added Parameter 3: int p2
-```
+Applying this on other entities that are not players or peds won't clamp the value.
```

Comment:
Are you sure this is true?

Reply:
Yeah, there's an if statement for ped entities and an else one for all other kind of entities.

![image](https://github.com/user-attachments/assets/534797cc-fb33-479e-91d6-39dcfbc3672d)

The else statement

![image](https://github.com/user-attachments/assets/5c45f69b-09f4-4785-9382-5f10b3978e6f)

Reply:
Confirmed whilst playing

![image](https://github.com/user-attachments/assets/d45300d9-1f55-4c8d-a3a1-406e04c75800)